tcptraceroute.yaml

```diff
@@ -54,4 +54,3 @@ subpackages:
 
 update:
   enabled: false
-  manual: true
```

Comment:
This one should remain as it is, as there is no update source connected for monitoring.

Reply:
Noted.